Sources/Mantis/CropViewController/CropViewController.swift

```diff
@@ -482,3 +609,67 @@ extension CropViewController {
         cropView.getExpectedCropImageSize()
     }
 }
+
+extension CropViewController: TransformDelegate {
+   
+    func updateEnableStateForUndo(_ enable: Bool) {
+        if config.enableUndoRedo {
+            delegate?.cropViewControllerDidUpdateEnableStateForUndo(enable)
+        }
+    }
+    
+    func updateEnableStateForRedo(_ enable: Bool) {
+        if config.enableUndoRedo {
+            delegate?.cropViewControllerDidUpdateEnableStateForRedo(enable)
+        }
+    }
+    
+    func updateEnableStateForReset(_ enable: Bool) {
+        if config.enableUndoRedo {
+            delegate?.cropViewControllerDidUpdateEnableStateForReset(enable)
+        }
+    }
+    
+    func getUndoManager() -> UndoManager {
+        return _undoManager
+    }
+    
+    func undo() {
+        if config.enableUndoRedo {
+            // Change State
+            if _undoManager.canUndo {
+                _undoManager.undo()
+            }
+        }
+    }
+    
+    func redo() {
+        if config.enableUndoRedo {
+            // Change State
+            if _undoManager.canRedo {
+                
```

Comment:
Remove unnecessary blank line to keep code consistent.

Reply:
change made.